.github/workflows/create-release.yml

```diff
@@ -0,0 +1,120 @@
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+
+name: Create Release
+
+jobs:
+  get_version:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Install poetry
+      run: |
+        python3 install-poetry.py --version 1.3.2
```

Comment:
Ok, introducing the use of poetry here. Not used this before. I'll have a look at it.